select.c

```diff
@@ -107,7 +107,7 @@ void select_loop(void) {
     FD_SET(netfd, &readfd);
     if(netfd >= maxfd) maxfd = netfd + 1;
 
-    if (mtrtype == IPPROTO_TCP)
+    if (mtrtype == IPPROTO_TCP || mtrtype == IPPROTO_SCTP)
```

Comment:
add the event handler(s) for socket? (I am not good at operating file descriptor)